auto_show/filters.py

```diff
@@ -0,0 +1,11 @@
+from django_filters import rest_framework as filters
+
+from .models import AutoShow
```

Comment:
don't use relative imports. use in all files:

from auto_show.models  import AutoShow

Reply:
++